Makefile

```diff
@@ -330,6 +338,7 @@ endif
 
 cocoa: $(BIN)/SameBoy.app
 quicklook: $(BIN)/SameBoy.qlgenerator
+xdg-thumbnailer: $(BIN)/XdgThumbnailer/sameboy-thumbnailer
```

Comment:
You might want to add `xdg-thumbnailer` to the `all` target if `FREEDESKTOP` is true. You probably also want to skip the `pkg-config` call on non-freedesktop targets as well (so it doesn't complain about missing libraries when building on e.g. macOS, even if you have pkg-config installed)